src/lib.rs

```diff
@@ -1317,6 +1317,15 @@ impl Node {
 				},
 			};
 
+			// insert channel's funding utxo into the wallet so we can later calculate fees
+			// correctly when viewing this splice-in.
+			self.wallet.insert_txo(funding_txo.into_bitcoin_outpoint(), funding_output).map_err(
+				|e| {
+					log_error!(self.logger, "Failed to splice channel: {:?}", e);
+					Error::ChannelSplicingFailed
+				},
+			)?;
```

Comment:
@tnull Don't know what the convention is here. Should we replace the `Error::PersistenceFailed` here with `Error::ChannelSplicingFailed`?